src/Tooltip.js

```diff
@@ -22,19 +31,29 @@ const afterStyles = {
   position: 'absolute',
   transform: 'translateX(-50%)',
   width: 0,
-}
+};
 
 const Tooltip = ({ emailAddress, className }) => (
-  <span
+  <div
     className={className + '__tooltip'}
     style={toolTipStyles}
   >
-    { emailAddress }
-    <span
+  <div className={className + '__tooltip-head'}>{ emailAddress }</div>
+  <div className={className + '__tooltip-body'}>
+    <button
+      className={className + '__tooltip--copy-email-address __tooltip-body-btn'}
+      style={toolTipBodyBtnStyles}
+      >Copy Email Address To ClipBoard</button>
```

Comment:
I prefer this code style:

```
<button
  className={className + '__tooltip--copy-email-address __tooltip-body-btn'}
  style={toolTipBodyBtnStyles}
>
  Copy Email Address To ClipBoard
</button>
```

I'll setup linting sometime in a new PR

Reply:
yes sure I will do for sure :) :+1: